static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -45,9 +65,9 @@ function UserActionsNavigator({event, crumbs}: Props) {
         {userActionCrumbs.map(item => (
           <PanelItemCenter
             key={item.id}
-            className={
+            className={`${
               currentUserAction && currentUserAction.id === item.id ? 'selected' : ''
-            }
+            } ${closestUserAction && closestUserAction.id === item.id ? 'onHover' : ''}`}
```

Comment:
Instead of using class name w/ a styled component, you can pass props to the styled component and use them. See https://emotion.sh/docs/styled#changing-based-on-props

It's similar to how we use theme. I would look around our codebase to see how it's used.

Reply:
alright @billyvg, tomorrow will iterate around those points you mention. Thanks for the suggestions regarding styled-components.